utils/build_swift/tests/test_driver_arguments.py

```diff
@@ -143,6 +143,13 @@ def test(self):
         return test
 
     @classmethod
+    def _generate_help_option_test(cls, option):
+        def test(self):
```

Comment:
This doesn't do anything, no? You aren't invoking test. What is the purpose of this?

Reply:
The meta-class here is used to generate unit-test cases dynamically. When Python is invoked with the `unittest` argument it will execute all methods on classes subclassing `unittest.TestCase` that begin with the word `test`. It's not required in any Python unittest code to invoke the test cases yourself.

Reply:
On a second look you're right, I forgot to return the actual test here. I'll fix this in the next PR